perun/postprocess/regression_analysis/tools.py

```diff
@@ -30,13 +30,9 @@ def validate_dictionary_keys(dictionary, required_keys, forbidden_keys):
     if not isinstance(dictionary, dict):
         raise exceptions.DictionaryKeysValidationFailed(dictionary, [], [])
     # Check all the required keys
-    for key in required_keys:
```

Comment:
Suggestion: Nice hack, though I would keep the previous non-compressed loop. It is well readable, simple, no need to compress it :). Same suggestion applies to line 37